### DISCUSSION TITLE
how can I use QGrid In intellij idea?I'm new no Node.js and Quasar CLI.I mean that Qgrid https://github.com/pratik227/quasar-qgrid

### DISCUSSION THREAD

**Discussion by Dozez13** [quasarframework/quasar#9301] [General - Components / Directives / etc] [answered]
_

**Comment by ontwikkelfabriek** [comment, marked as answer]
1) click on the tab 'terminal' (usually somewhere at the bottom left). 
2) make sure you are in the root of your Quasar project.( where your package.json resides)
3) run this command in the terminal to install the extension: `quasar ext add qgrid`
4) now QGrid is available to be used in your project.(See the examples of Qgrid)

**Reply by Dozez13** [reply]
Do I need to create package.json myself? I just do not know how to work with Node.js an Quasar CLI

**Reply by ontwikkelfabriek** [reply]
> Do I need to create package.json myself? 

this file is created when you create a Quasar app with the Quasar CLI:

> I just do not know how to work with Node.js an Quasar CLI

are you able to create a new Quasar project ( in windows cmd for example) with Quasar CLI?

**Reply by Dozez13** [reply]
Thanks for the answer. I didn’t specify that I use Java and servlets, and I don’t know how to connect Quasar extension

**Comment by TobyMosque** [comment]
Based in what u said, I think u're using the UMD version of the quasar.
If that is the case, you'll need to ask @pratik227 to support UMD.

https://next.quasar.dev/app-extensions/development-guide/introduction#getting-started

**Reply by pratik227** [reply]
Yes @TobyMosque It's not built for the UMD version. I will publish it soon.
